Libraries/LibJS/Token.cpp

```diff
@@ -155,6 +155,9 @@ String Token::string_value(StringValueStatus& status) const
             case '\\':
                 builder.append('\\');
                 break;
+            case '\n':
+                status = StringValueStatus::OkWithLineContinuation;
```

Comment:
Do we need a new state for this, why not simply `Ok`?

Reply:
Oh... it was to prevent
```
"use\
 strict"
``` 
from working but now I see that there is another mechanism for ensuring that. 
I'll make some changes.

Reply:
oops I changed that recently, forgot to update the issue mentioning it - sorry about that :)

Reply:
Haha, all good. Didn't notice your check when rebasing onto master - it made the additional state obsolete so I'm removing that.